source/Trashbot.js

```diff
@@ -17,4 +17,9 @@ export default class Junkership extends Pixi.Sprite {
             this.position.x = Reference.GAME_WIDTH
         }
     }
+    onCollision(){
+        game.removeChild(this)
+        this.destroy()
+        game.score++
```

Comment:
That also covers the problem of determining whether a Projectile came form a Junkership or from a Turret/Sniper Trashbot, since the former won't harm players but the latter can. Changed in next commit.